Slides/estimation-slides.Rmd

```diff
@@ -431,7 +431,7 @@ $SE(\widehat{ATE}) = \sqrt{\sum^J_{j=1} (\frac{N_{j}}{N})^2SE^2(\widehat{ATE}_j)
 
 ## Estimating the ATE in block-randomized experiments
 
-One approach to estimation simply replaces $ATE_j$ with $\widehat{ATE}$ above:
+One approach to estimation simply replaces $ATE_j$ with $\widehat{ATE}$ below:
```

Comment:
Is this correct?

There is another sentence 
One approach to estimation simply replaces $ATE_j$ with $\widehat{ATE}$ above:

Reply:
no, it was correct before